content/homework.rst

```diff
@@ -38,6 +50,17 @@ There is nobody to tell you when to work, how to work or to check that you
 are at work. It is your own style to work and you need to find ways to
 improve the way you work.
 
+The sustainable work hours number varies considerably by person so take the time to observer your limits.
+
+Define and respect boundaries between work and your personal life.
+Realize your work throughput limit and stop working as soon as
+you hit the limit. It will be best for your and the project.
+
+There is no correlation between work hours and experience level and
+working hours is a personal choice.
```

Comment:
The conjunctions makes it hard to infer which are on one side and which on the other side. Perhaps: _between work hours and experience level on one side and working hours on the other side_? I still don't think I get the idea.